### PR TITLE
Feature/asv 1586 change poa appc to appc credits

### PR DESCRIPTION
**Pull request by paquiquinho**
**What does this PR do?**

This PR aims to change the copy in the PoA element of the appview to say AppCoins Credits instead of Appcoins. This came with a slight redesign to that element

**Database changed?**

No

**Where should the reviewer start?**

- [ ] appview_appc_reward.xml

**How should this be manually tested?**

Check the element and make sure everything is good

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1586](https://aptoide.atlassian.net/browse/ASV-1586)

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

### CODE REVIEW

File: app/src/main/java/cm/aptoide/pt/app/view/AppViewFragment.java [hunk 1]
```diff
@@ -1592,18 +1591,20 @@ private SpannableString formatAppCoinsRewardMessage() {
...
     } else {
       tryAppMessage =
           getResources().getString(R.string.appc_message_appview_appcoins_reward, reward);
       spannable = new SpannableString(tryAppMessage);
     }
 
-    spannable.setSpan(new ForegroundColorSpan(getResources().getColor(StoreTheme.get(theme)
-            .getPrimaryColor())), tryAppMessage.indexOf(reward),
-        tryAppMessage.indexOf(reward) + reward.length(), Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
+    spannable.setSpan(
```
Comment (phillipsgoncalves) [inline app/src/main/java/cm/aptoide/pt/app/view/AppViewFragment.java]: Why did we stop using the theme here?

File: app/src/main/res/layout-land/appview_appc_reward.xml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        android:layout_marginLeft="22dp"
+        android:layout_marginStart="22dp"
+        android:layout_marginTop="16dp"
+        android:src="@drawable/ic_appcoins"
+        />
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/top_line"
```
Comment (phillipsgoncalves) [inline app/src/main/res/layout-land/appview_appc_reward.xml]: You are using layout_below in a Linear layout, which is an unsupported attribute for linear layout.

File: app/src/main/res/layout-land/appview_appc_reward.xml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  <LinearLayout
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_centerHorizontal="true"
+      >
+    <ImageView
+        android:id="@+id/appcoins_logo"
+        android:layout_width="48dp"
+        android:layout_height="48dp"
+        android:layout_below="@id/top_line"
```
Comment (phillipsgoncalves) [inline app/src/main/res/layout-land/appview_appc_reward.xml]: You are using layout_below in a Linear layout, which is an unsupported attribute for linear layout.

File: app/src/main/res/layout-land/appview_appc_reward.xml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+          style="@style/Aptoide.TextView.Medium.XXS"
+          />
+    </LinearLayout>
+  </LinearLayout>
+
+  <View
+      android:layout_width="match_parent"
+      android:layout_height="1dp"
+      android:background="#f6f7f9"
+      app:layout_constraintBottom_toBottomOf="parent"
```
Comment (phillipsgoncalves) [inline app/src/main/res/layout-land/appview_appc_reward.xml]: You are using these constraints, but it is a relative layout not a constraint

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -542,8 +542,8 @@
   <string name="appc_home_bundle_iab">Use Your AppCoins</string>
   <string name="appc_home_bundle_poa">Earn AppCoins Credits</string>
   <string name="appc_title_about_appcoins">About AppCoins</string>
-  <string name="appc_message_appview_appcoins_reward">Try this app and get %s</string>
-  <string name="appc_message_appview_appcoins_reward_with_value">Try this app for 2 minutes and get %1$s %2$s</string>
+  <string name="appc_message_appview_appcoins_reward">Try this app for 2 minutes and get\n%s</string>
```
Comment (phillipsgoncalves) [inline app/src/main/res/values/strings.xml]: same changes in the strings file as the other pull request. Did Carlos review this?

File: app/src/main/java/cm/aptoide/pt/app/view/AppViewFragment.java (hunk 1 above)
Comment (paquiquinho) [reply app/src/main/java/cm/aptoide/pt/app/view/AppViewFragment.java]: the color wasn't the same

**Pull request merged**